Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,21 +1,77 @@
 import React from 'react';
+import CalculatorDisplay from "./components/DisplayComponents/CalculatorDisplay";
+import NumberButton from "./components/ButtonComponents/NumberButton";
+import ActionButton from "./components/ButtonComponents/ActionButton";
 import './App.css';
 
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className="app">
+      <CalculatorDisplay />
+      <ActionButton 
+      buttonStyles="white"
+      text="Clear"
+      />
+      <NumberButton 
+      buttonStyles="red"
+      text="÷"
+      />
+      <NumberButton 
+      buttonStyles="white"
+      text="7"
+      />
+      <NumberButton 
+      buttonStyles="white"
+      text="8"
+      />
+      <NumberButton 
+      buttonStyles="white"
+      text="9"
+      />
+      <NumberButton 
+      buttonStyles="red"
+      text="×"
+      />
+      <NumberButton 
+      buttonStyles="white"
+      text="4"
+      />
+      <NumberButton 
+      buttonStyles="white"
+      text="5"
+      />
+      <NumberButton 
+      buttonStyles="white"
+      text="5"
+      />
+      <NumberButton 
+      buttonStyles="red"
+      text="-"
+      />
+      <NumberButton 
+      buttonStyles="white"
+      text="1"
+      />
+      <NumberButton 
+      buttonStyles="white"
+      text="2"
+      />
+      <NumberButton 
+      buttonStyles="white"
+      text="3"
+      />
+      <NumberButton 
+      buttonStyles="red"
+      text="+"
+      />
+      <ActionButton 
+      buttonStyles="white"
+      text="0"
+      />
+      <NumberButton 
+      buttonStyles="red"
+      text="="
+      />
```

Comment:
Most people put these in one line like <NumberButton buttonStyles='white' text='0' /> It doesn't matter either way but I tend to do this sometimes because it makes it a little more readable for me. Good job for thinking outside of the box and doing it a little differently than the lecture!